backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/projection/impl/GeoPointFieldProjectionBuilderFactory.java

```diff
@@ -63,4 +72,15 @@ public boolean isDslCompatibleWith(LuceneFieldProjectionBuilderFactory obj) {
 		return codec.isCompatibleWith( other.codec ) &&
```

Comment:
I think the "isCompatible" method should take the "projectable" field into acccount. Otherwise you run the risk of the field being projectable in one index and not the other, and allowing projection anyway. Maybe we should also adapt the exception message to mention that it can happen when the field is not projectable in every index.

Same goes for all the other factories, obviously.

Reply:
Right, fixed.

At the moment, the message is very vague. Not a big fan of making it too specific while a lot of things move every day. Better vague than wrong. Maybe let's wait for some feedback from the field.